core/css/guest.css

```
@@ -592,8 +591,8 @@ form .warning input[type='checkbox']+label {
 
 .alternative-logins a.button:focus,
 .alternative-logins li a:focus {
-	border: 2px solid var(--color-primary-hover);
-	background-image: linear-gradient(40deg, var(--color-primary) 0%, var(--color-primary-light) 100%);
+	border: 2px solid black !important;
+	background-color: var(--color-primary-element-light) !important;
```

```suggestion
	background-color: var(--color-primary-element-hover) !important;
```

Should make it compliant with the contrast ratio.